lib/internal/util/inspect.js

```diff
@@ -1306,25 +1409,24 @@ function formatError(err, constructor, tag, ctx, keys) {
     stack = `[${stack}]`;
   } else {
     let newStack = stack.slice(0, stackStart);
-    const lines = getStackFrames(ctx, err, stack.slice(stackStart + 1));
+    const stackFramePart = stack.slice(stackStart + 1);
+    const lines = getStackFrames(ctx, err, stackFramePart);
     if (ctx.colors) {
       // Highlight userland code and node modules.
-      for (const line of lines) {
+      const workingDirectory = safeGetEscapedCWD(stackFramePart);
+      for (let line of lines) {
         const core = line.match(coreModuleRegExp);
         if (core !== null && NativeModule.exists(core[1])) {
           newStack += `\n${ctx.stylize(line, 'undefined')}`;
         } else {
-          // This adds underscores to all node_modules to quickly identify them.
-          let nodeModule;
           newStack += '\n';
-          let pos = 0;
-          while ((nodeModule = nodeModulesRegExp.exec(line)) !== null) {
-            // '/node_modules/'.length === 14
-            newStack += line.slice(pos, nodeModule.index + 14);
-            newStack += ctx.stylize(nodeModule[1], 'module');
-            pos = nodeModule.index + nodeModule[0].length;
+
+          line = markNodeModules(ctx, line);
+          if (workingDirectory !== undefined) {
+            line = markCwd(ctx, line, workingDirectory);
```

Comment:
```suggestion
            line = markCwd(ctx, line, workingDirectory[1]);
            line = markCwd(ctx, line, workingDirectory[0]);
```

Reply:
This would cause a double escaping in the common esm case (no escaping required). It would not really hurt as it does not change anything in the visual output but I still think it's not ideal. I went ahead and updated the code to work with a ESM / CJS mixture as well without double escaping.